Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -382,8 +382,11 @@ - (void)performFinalCleanup {
 
     // Enable 'undo' if this plug-in is being unregistered
     if (self.shouldEnableUndoUponUnregistration) {
-        [self.parentTextView.undoManager enableUndoRegistration];
+        if(!self.parentTextView.undoManager.isUndoRegistrationEnabled)
```

Comment:
Please add this to the existing if statement as an "and" condition and please follow the style around you.

Also please add a missing unit test capturing the crash that you're fixing. Thanks!